Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM quay.io/konveyor/builder:v1.17.2 as builder
+FROM quay.io/konveyor/builder as builder
```

Comment:
Should we be using a tagged version such that it is clear what go version is used?